include/assimp/color4.h

```diff
@@ -63,8 +63,7 @@ class aiColor4t
     aiColor4t (TReal _r, TReal _g, TReal _b, TReal _a)
         : r(_r), g(_g), b(_b), a(_a) {}
     explicit aiColor4t (TReal _r) : r(_r), g(_r), b(_r), a(_r) {}
-    aiColor4t (const aiColor4t& o)
-        : r(o.r), g(o.g), b(o.b), a(o.a) {}
+    // aiColor4t (const aiColor4t& o) = default;
```

Comment:
Dead code.

Reply:
Do you prefer the copy constructor to be explicitly default or implicitly default? It can be either removed or uncommented.

Reply:
I think either C++17 or C++20 will change it so that if we don't explicitly default the copy constructor it will no longer be implicitly emitted. GCC already has a warning about this. We should add either default or explicitly write all the constructors for future proofing.

Reply:
Ok, I didn't know that. The whole point of this is to make these classes trivially copiable, so I'd go the explicitly defaulted way.
```suggestion
    aiColor4t (const aiColor4t& o) = default;
```